eng/scripts/Fetch-Namespaces-DotnetNupkg.ps1

```diff
@@ -0,0 +1,42 @@
+<#
+  .SYNOPSIS
+  Fetch the namespaces from dotnet nupkg and checkin to docs/metadata
+
+  .DESCRIPTION
+  The scripts is used for docs.ms to fetch namespace list for particular nupkg artifact.
+
+  .PARAMETER DotnetNupkgLocation
+  Specifies the fo nupkg location.
+
+  .PARAMETER DocRepoLocation
+  Specifies Location of the root of the docs.microsoft.com reference doc location. 
+
+  .PARAMETER ArtifactName
+  The artifact name. E.g. Azure.Template
+#>
+[CmdletBinding()]
+param (
+  [string] $DotnetNupkgLocation = "",
+  [string] $DocRepoLocation = "",
+  [string] $ArtifactName = ""
```

Comment:
Nit: You don't need to pass `= ""`. That's already the default behavior for strings in PowerShell.